react/cra-template/template/public/index.html

```diff
@@ -20,6 +20,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
I am honestly a little torn on this.  This is super simple but does have the potential disadvantage that we are dependent on the external cdn for the app to look right (google _should_ be up but what if it isn't).

Thoughts on using something like this https://github.com/fontsource/fontsource to include these fonts in the bundle?

//fonts.ts
```
import "fontsource-roboto";
import "fontsource-material-icons";
```
theme.ts
```
import "./fonts";

// do material theme things
```